### DISCUSSION TITLE
Directus won't run on Firefox and Edge, but is OK on Samsung Internet

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#16429] [Archive]
on Samsung Internet, everything is OK. I can connect, edit and save.

on Edge, I get an `ERR_SSL_PROTOCOL_ERROR`
and on Firefox, Error code: `SSL_ERROR_RX_RECORD_TOO_LONG`

My server is a Debian 11.5 

Do you have any resources about this problem even it looks like not to be at first a Directus problem?

**Comment by br41nslug** [comment]
As Directus does not handle SSL out-of-the-box looks like SSL is misconfigured on your proxy, make sure the latest SSL protocols required bu modern browsers are supported.

**Reply by Pachat** [reply]
@br41nslug 
Thanks for your suggestion.
Do you mean that Directus cannot send and receive encrypted requests, or that is has to be settled outside of Directus?
I now have SSL working on all browsers. But only on ports 80 and 443. May be due to Let's Encrypt limitations?

**Comment by Kvarkas** [comment]
having same problem with latest Directus and ubuntu server (running from docker) page response nice with ssl for all except directus
